features/css-clip-path.md

```diff
@@ -0,0 +1,12 @@
+---
+title: "CSS clip-path"
+category: css
+bugzilla: 1247229
+firefox_status: 54
+mdn_url: https://developer.mozilla.org/Web/CSS/clip-path
+spec_url: https://drafts.fxtf.org/css-masking-1/#the-clip-path
+caniuse_ref: clip-path
```

Comment:
The issue is that this feature doesn't exist on caniuse. Perhaps you meant `css-clip-path`?